lib/test-requirements.txt

```diff
@@ -1,7 +1,7 @@
 # Packages used to test things users may do with Streamlit.
 # As of October 22, 2021, only the latest version of Bokeh (2.4.1) is supported.
 # NOTE: Python Bokeh and BokehJS versions must always match.
-bokeh==2.4.1
+bokeh>=2.4.1
```

Comment:
It might be better to update this to: `bokeh==2.4.3` since this is only the test requirement. And maybe remove the ` As of October 22, 2021..` line from the comment above.

Reply:
I agree if we were to bump this, it would bokeh==2.4.1.

I think we should update the comment to 
`As of June 16, 2022, only the latest version of 2.4.3 is supported.` or something like that. @LukasMasuch , what do you think? I recall adding that line on purpose because it seems weird to pin a specific version.

Reply:
I think the next line is already giving a reason for the pinning, maybe we can update it to:

`# NOTE: Python Bokeh and BokehJS versions must always match (currently 2.4.3).`

but I'm fine with either way :)